hi-ml-azure/testazure/testazure/test_himl.py

```diff
@@ -119,6 +120,103 @@ def test_write_run_recovery_file(mock_run: mock.MagicMock) -> None:
     assert expected_run_recovery_id != recovery_file_text
 
 
+@pytest.fixture(scope="module")
+def dummy_max_num_nodes_available() -> int:
+    return randint(2, 10)
+
+
+@pytest.fixture(scope="module")
+def dummy_compute_cluster_name() -> str:
+    return 'dummy_cluster'
+
+
+@pytest.fixture(scope="module")
+def mock_scale_settings(dummy_max_num_nodes_available: int) -> MagicMock:
+    return MagicMock(maximum_node_count=dummy_max_num_nodes_available)
```

Comment:
Maybe add comments here and below what you are trying to mock here? (like: Mocking a ComputeCluster object which has a property maximum_node_count...)